### DISCUSSION TITLE
Only three thumbnails show on thumbnail bar

### DISCUSSION THREAD

**Discussion by ro345** [mifi/lossless-cut#2029] [Q&A]
I run lossless cut on several PCs.  On one of my PCs only three thumbnails ever show up on the thumbnail task bar.  My other PCs work correctly, showing 10 or so thumbnails.  I cant figure out what the difference is.  Files are being loaded from local disk.  PC has plenty of memory and CPU.  This happens with every file on this PC.  I have tried dozens of files on this same PC and they always show 3 thumbnails.  Any ideas?

<img width="1274" alt="image" src="https://github.com/mifi/lossless-cut/assets/22797392/fc78aa42-1457-4024-814c-9e255f9085be">

**Comment by mifi** [comment]
Hi. I believe you're being hit by this logic:

https://github.com/mifi/lossless-cut/blob/9c9c8639acf6a752300d56f3c5d5ef9ee9f1cb60/src/renderer/src/ffmpeg.ts#L540

explanation:

it tries to render as many thumbnails as it can within 3 seconds. if it can render 3 or less, it will settle with 3. this could indicate that that computer is a bit slower than the others? maybe the logic should be changed, but the thought behind it was that it shouldn't overwhelm the cpu with thumbnail rendering if it's taking too long

**Reply by ro345** [reply]
Interesting, thanks for that info.  I'm sure that must be it.  Must be something specifically with that PC, but it has 12 cores and 64GB RAM.  I will need to look further.
